app/services/state_file/after_transition_messaging_service.rb

```diff
@@ -0,0 +1,48 @@
+module StateFile
+  class AfterTransitionMessagingService
+    include Rails.application.routes.url_helpers
+
+    def initialize(intake)
+      @intake = intake
+      raise(ArgumentError, "Unsupported intake type: #{intake.class.name}") unless %w[StateFileAzIntake StateFileNyIntake].include?(intake.class.name)
+    end
+
+    def send_efile_submission_accepted_message
+      if @intake.calculated_refund_or_owed_amount.positive?
+        message = StateFile::AutomatedMessage::AcceptedRefund
+        body_args = { return_status_link: return_status_link }
+      else
+        message = StateFile::AutomatedMessage::AcceptedOwe
+        body_args = { state_pay_taxes_link: state_pay_taxes_link, return_status_link: return_status_link }
+      end
+
+      messaging_service = StateFile::MessagingService.new(
+        intake: @intake,
+        message: message,
+        body_args: body_args)
+
+      messaging_service.send_message
+    end
+
```

Comment:
Ditto

Reply:
yes @rickreyhsig was gonna add this in for the subsequent story